src/sound/manager.js

```diff
@@ -31,6 +31,13 @@ class SoundManager extends EventHandler {
          * @private
          */
         this._context = null;
+        /**
+         * The current state of the underlying AudioContext.
+         *
+         * @type {string}
+         * @private
+         */
+        this._state = 'not created';

```

Comment:
please create some constant states just before `class SoundManager` line, something along:
```
const STATE_NOTCREATED = "not created";
...
```

and use those in the code to avoid possibility of typo.